### ISSUE TITLE
E.T.A. for new features / versions

### ISSUE THREAD

**Issue by FZ-Iskandar** [cgaspard/mantiskanban#40]
I think that it would be for the interest of eveyone to have a more or less regular update on when some features are expected or what are you focusing on. I've seen another post with ideas but there is no ETA for any of this features... the only hint on that is seeing what files have been updated here in github or visiting the demo and see if you find something new.

In fact, the best choice would be adding a page on your demo web page to let people know about news such as new cool features, bug fixes and so on.

Another aproach would be to use the roadmap on the demo web page the same way they do it at the mantisbt web (https://www.mantisbt.org/bugs/roadmap_page.php).

And finally, if the other two options seems to you as too much trouble/work at least you could post here every now and then what are you working on at the moment and when we should espect something to be released.

I know that this is a free project that you probably do on your spare time, so I don't want to put any preassure on you, but i think that having such a great mantis add-on without almost any information on it's development or manuals / FAQS is a pity. ;-)

**Comment by mtonina** [comment]
Hi guys. This tool is awesome. I need to know if this project is alive, and if in the future will be updates since MantisBT is always under improvements. Thanks

**Comment by cgaspard** [comment]
Yeah its alive, but work schedule doesn’t give me much free time lately.  Been a year or more since I’ve been able to give significant time to it.   I tried to get the mantis guys to integrate the tool directly, but they didn’t seem interested.

Eventually I’ll probably make  a stand alone version since the Kanban itself can be plugged into any backend with the right APIs.

If I do that,’ll leave this version around for anyone who wants to stick with mantis.

Corey
On June 18, 2015 at 11:25:10 AM, mtonina (notifications@github.com) wrote:

Hi guys. This tool is awesome. I need to know if this project is alive, and if in the future will be updates since MantisBT is always under improvements. Thanks
